api/pkg/db/migration/202008121703_add_user_resource_rating_table.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2020 The Tekton Authors.
```

Comment:
Can we add this migration in `202006071000_add_resource_and_resource_versions_tables.go` file instead of adding a separate migration as it just creates the table for `userResourceRating` and adding foreign key constraints  which is done there as well???

Reply:
Existing migration is never changed or updated. Migration runs only once. If you have an existing instance of DB when you run a migration, its ID is saved in the migration table, once ID is inserted that migration is not run again. So, adding anything in previous migration won't have any effect.